### PR TITLE
Updated Log Symptoms Page according to new designs

### PR DESCRIPTION
**Pull request by yichenesia**
# Description

- Save button always there overlaid on top 
- Header should stay there overlaid 

Screen Shot 2022-04-26 at 5 24 45 PM

- Shift screen up so you can see what you’re typing in notes
Screen Shot 2022-04-26 at 5 25 07 PM

**ONLY ISSUE** The status bar at the top is now white. I'm sorry little one but there was no other way 😭 .... What did it cost....everything..... ;-;

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if required)
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: tpp-app/src/home/pages/LogSymptomsScreen.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ import { TRACK_SYMPTOMS } from "../../services/utils/constants";
 import { CALENDAR_STACK_SCREENS } from "../CalendarNavigator";
 import { getISODate } from '../../services/utils/helpers';
 import { calculateAverages } from "../../services/CalculationService";
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view'
```
Comment (faye-tan) [inline tpp-app/src/home/pages/LogSymptomsScreen.js:16]: need to push package.json and package-lock.json as well

Comment (faye-tan) [inline tpp-app/src/home/pages/LogSymptomsScreen.js:16]: need to push package.json and package-lock.json as well

**Review by faye-tan — CHANGES_REQUESTED** [review]
just push package.json and package-lock.json and should be good! 👍

File: tpp-app/src/home/pages/LogSymptomsScreen.js [hunk 2]
```diff
@@ -279,7 +280,8 @@ export default function LogSymptomsScreen({ navigation, route }) {
 
 
   return (
-    <SafeAreaView style={styles.screen}><ScrollView style={styles.content}>
+    <SafeAreaView style={styles.screen}>
+    {/* <ScrollView style={styles.content}> */}
```
Comment (faye-tan) [inline tpp-app/src/home/pages/LogSymptomsScreen.js:284]: just delete if we don't need

File: tpp-app/src/home/pages/LogSymptomsScreen.js [hunk 3]
```diff
@@ -338,31 +341,43 @@ export default function LogSymptomsScreen({ navigation, route }) {
...
+            onPress={() => {
+              if (!isDirty) return; // if no changes, do nothing
+              setSubmitting(true);
+            }}
+          >
+            <Text style={{color: '#fff'}}>Save</Text>
+          </TouchableOpacity>
+        </View>
       </View>
+    {/* </ScrollView> */}
```
Comment (faye-tan) [inline tpp-app/src/home/pages/LogSymptomsScreen.js:360]: same here

File: tpp-app/src/home/pages/LogSymptomsScreen.js (hunk 1 above)
Comment (yichenesia) [reply tpp-app/src/home/pages/LogSymptomsScreen.js:16]: Bruh why didn't they pushhh 

thx for catching it ><

**Review by faye-tan — APPROVED** [review]

**Pull request merged**